internal/provider/schema_cloudflare_worker_script.go

```diff
@@ -55,6 +55,23 @@ var webAssemblyBindingResource = &schema.Resource{
 	},
 }
 
+var serviceBindingResource = &schema.Resource{
```

Comment:
this isn't used or referenced in the schema so you're unable to actually use it. you'll need to reference it below like the other bindings.

Reply:
Oh pff, I knew I was missing something